main/VdaggerV.cpp

```diff
@@ -218,36 +230,34 @@ int main(int ac, char *av[]) {
               });
           if (!(it != vdaggerv_lookup.end())) {
             vdaggerv_lookup.emplace_back(
-                VdaggerVQuantumNumbers(id, momentum, displacement));
+                id, momentum, displacement);
             id++;
           }
         }
+  }
   OperatorLookup operator_lookuptable;
   operator_lookuptable.vdaggerv_lookup = vdaggerv_lookup;
   operator_lookuptable.index_of_unity = 0;
 
+
   // Loop over all configurations stated in the infile -------------------------
-  for (ssize_t config_i = start_config; config_i <= end_config;
-       config_i += delta_config) {
+  for (ssize_t config_i = gd.start_config; config_i <= gd.end_config;
+       config_i += gd.delta_config) {
     std::cout << "\nprocessing configuration: " << config_i << "\n\n";
+    OperatorFactory meson_operators(gd.Lt,
+                                  gd.Lx,
+                                  gd.Ly,
+                                  gd.Lz,
+                                  gd.number_of_eigen_vec,
+                                  (gd.quarks)[0].number_of_dilution_E,
+                                  gd.operator_lookuptable,
+                                  gd.handling_vdaggerv,
+                                  gd.path_vdaggerv,
+                                  gd.path_config,
+                                  gd.hyp_parameters);
+
+    meson_operators.build_vdaggerv(gd.filename_eigenvectors, config_i, gd);
```

Comment:
does this now work without defining any operators or correlators?

Reply:
yes, only the quarks have to be defined

Reply:
How does that make sense? VdaggerV is independent of the perambulators. The only thing that should have to be specified are the details about the eigensystems and the maximum  \vec{p}^2 class to be considered.

Reply:
ping?!